example/src/examples/BuildingFullInfo.tsx

```diff
@@ -13,29 +14,29 @@ export const BuildingFullInfo = () => {
   const [outdoorPOIs, setOutdoorPOIs] = useState<any>();
   const [building, setBuilding] = useState<any>();
 
```

Comment:
O código pareceme máis leible antes, pode ser por influencia de ios. De primeiras a min b, f ou ip  non me din demasiado, inda que polo contexto se poden intuir.